AsyncWeb/Controllers/RoomsController.cs

```diff
@@ -103,34 +103,42 @@ public async Task<IActionResult> DeleteRoom(int id)
             return NoContent();
         }
 
-        private bool RoomExists(int id)
-        {
-            return _context.Rooms.Any(e => e.Id == id);
-        }
-
-
-        // ====== Adding Amenities to Hotel ======
-
-
-
-
-
-
-
 
 
 
+        // ====== Adding Amenities to Hotel ======
 
+        // TODO: Create 2 routes. 1 for POST and 1 for DELETE
+        [Route("{roomId}/Amenity/{amenityId}")] // check keiths code on the repo
 
 
+        [HttpPost]
+        public async Task<ActionResult> PostAmenity(Amenities amenity)
```

Comment:
Mismatch between the route values (`roomId` and `amenityId`) and the action parameters (just `amenity`). You want:

```csharp
PostAmenity(int roomId, int amenityId)
```